pegasus/sites.v3/code.org/views/learn_carousels.haml

```diff
@@ -1,21 +1,46 @@
-- learn_categories = []
-- learn_categories << {tag:"JavaScript", name: I18n.t(:carousel_heading_javascript)}
-- learn_categories << {tag:"Beginner", name: I18n.t(:carousel_heading_beginners)}
-- learn_categories << {tag:"Unplugged", name: I18n.t(:carousel_heading_unplugged)}
-- learn_categories << {tag:"Apps", name: I18n.t(:carousel_heading_devices)}
-- learn_categories << {tag:"OtherLang", name: I18n.t(:carousel_heading_languages)}
-- learn_categories << {tag:"Teacherled", name: I18n.t(:teach_led_lesson_plans)}
-
-- beyond_categories = []
-- beyond_categories << {tag:"Beginner", name: I18n.t(:carousel_heading_beyond_beginners)}
-- beyond_categories << {tag:"JavaScript", name: I18n.t(:carousel_heading_beyond_javascript)}
-- beyond_categories << {tag:"Apps", name: I18n.t(:carousel_heading_beyond_devices)}
-- beyond_categories << {tag:"OtherLang", name: I18n.t(:carousel_heading_beyond_languages)}
-- beyond_categories << {tag:"MakeApps", name: I18n.t(:carousel_heading_apps_games)}
-- beyond_categories << {tag:"Univ", name: I18n.t(:carousel_heading_university)}
-- beyond_categories << {tag:"Web", name: I18n.t(:carousel_heading_webpages)}
-- beyond_categories << {tag:"Robot", name: I18n.t(:carousel_heading_robots)}
-- beyond_categories << {tag:"IDE", name: I18n.t(:carousel_heading_ide)}
+- all_categories = {}
```

Comment:
Might be more readable with a single `:ruby` block instead of `-` for every line.  (Of course, even better is extacting to a separate `.rb` file.